compatibility/compatibility-tests/src/test/java/org/projectnessie/tools/compatibility/tests/AbstractCompatibilityTests.java

```diff
@@ -56,16 +57,30 @@
 @VersionCondition(maxVersion = Version.NOT_CURRENT_STRING)
 public abstract class AbstractCompatibilityTests {
 
-  public static final String NESSIE_0_30_0 = "0.30.0";
   @NessieAPI protected NessieApiV1 api;
   @NessieAPI protected NessieApiV2 apiV2;
   @NessieVersion Version version;
 
   abstract Version getClientVersion();
 
+  Branch createMissingNamespaces(Branch branch, ContentKey namespaceKey)
+      throws NessieNotFoundException {
+    for (int i = 1; i <= namespaceKey.getElementCount(); i++) {
+      try {
+        api.createNamespace()
+            .refName(branch.getName())
+            .namespace(Namespace.of(namespaceKey.getElements().subList(0, i)))
+            .create();
```

Comment:
Would it be worth exploring the idea of a create method that also creates all the parent namespaces? The equivalent of mkdir -p so to speak.

Reply:
Depends on whether it would be useful for users.